src/scales/scale.category.js

```diff
@@ -33,86 +33,92 @@ module.exports = Scale.extend({
 	buildTicks: function() {
 		var me = this;
 		var labels = me._getLabels();
+		var minIndex = me.minIndex;
+		var maxIndex = me.maxIndex;
+
 		// If we are viewing some subset of labels, slice the original array
-		me.ticks = (me.minIndex === 0 && me.maxIndex === labels.length - 1) ? labels : labels.slice(me.minIndex, me.maxIndex + 1);
+		var ticks = me.ticks = (minIndex === 0 && maxIndex === labels.length - 1) ? labels : labels.slice(minIndex, maxIndex + 1);
+
+		if (me.options.ticks.reverse) {
+			me.ticks = ticks.slice().reverse();
+		}
+	},
+
+	convertTicksToLabels: function() {
+		var me = this;
+
+		me._tickValues = me.ticks.slice();
```

Comment:
can we either make this not necessary or add a comment explaining why it's needed? it's confusing to me that we would store ticks in both `ticks` and `_tickValues`

Reply:
Comment added. The labels need to be stored because they can be modified by the user callback while `getPixelForValue` requires the original label value.